CapitalForecasting/Modules/hierarchical_modeling.R

```diff
@@ -4,6 +4,7 @@ library(fable)
 library(feasts)
 library(lubridate)
 
+# sample data
 df <-  read_csv("C:/Users/AxelTorbenson/OneDrive - eCapital Advisors, LLC/Documents/templates/CapitalForecasting/Data/working_capital_fake_data.csv")
 
 create_tsibble <- function(data, date) {
```

Comment:
Can you put line 10 into line 11's pipe?

Reply:
That line is redundant, I'll delete it